common/db.py

```diff
@@ -315,30 +312,31 @@ def update_locked_batches(self, app_name: str, sig_data: dict[str, Any]) -> None
             return
         batches: dict[str, Any] = self.apps[app_name]["batches"]
         # fixme: sort should be removed after updating batches dict to list
-        for batch in sorted(list(batches.values()), key = lambda batch: batch.get("index", 0)):
+        for batch in sorted(list(batches.values()), key=lambda batch: batch.get("index", 0)):
             if "index" in batch and batch["index"] <= sig_data["index"] and batch["state"] != "finalized":
                 batch["state"] = "locked"
         if not batches.get(sig_data["hash"]):
             return
         target_batch: dict[str, Any] = batches[sig_data["hash"]]
         target_batch["lock_signature"] = sig_data["signature"]
         target_batch["locked_nonsigners"] = sig_data["nonsigners"]
+        target_batch["tag"] = sig_data["tag"]
```

Comment:
it should be "locked_tag" just like "locked_nonsigners" to not conflict with finalizing tag if they are different.